### DISCUSSION TITLE
Customize the "azure AD" authentication sign in button in the grafana login form

### DISCUSSION THREAD

**Discussion by bharath8593** [grafana/grafana#39033] [General]
Hey

We have installed grafana on our environment via helm charts and enabled "auth.azuread" authentication type.  So if we visit to "https://<hostname>/grafan/login", we are seeing the login form with the button as **"Sign in with Microsoft"**. But we would like to change that login button **to "Sign in COMPANY NAME SSO"**. Is this possible and can you please give us some suggestions ?  thanks in advance

https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack

Please find the application details
Kube prometheus chart = 18.0.2
Grafana version = 8.1.1

Please let me know if you require additional information on the same. Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
